jeeves/blockers.py

```diff
@@ -152,9 +152,7 @@ def get_other_blockers(blockers, job_name):
 		returns list of 'other' blockers
 	'''
 	other = []
-	other_blockers = blockers[job_name].get('other')
-	if other_blockers is None:
-		return other
+	other_blockers = blockers[job_name]['other']
```

Comment:
What is the motivation behind this change?

Reply:
Makes sense; the behavior appears to be the same either way so this is fine with me